packages/carbon-graphs/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # ChangeLog
 
 ## Unreleased
+* Fixed
+  * Added prop `rotateAngle` to allow rotation of x-axis tick labels to prevent overlap.
```

Comment:
```suggestion

* Fixed
  * Added prop `rotateAngle` to allow rotation of x-axis tick labels to prevent overlap.
```
Can we add space for consistency